hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/utils/TestUtils.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigtable.hbase.replication.utils;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Iterator;
+import java.util.List;
+import java.util.NavigableMap;
+import org.apache.hadoop.hbase.Cell;
+import org.apache.hadoop.hbase.CellUtil;
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Mutation;
+import org.apache.hadoop.hbase.client.Result;
+import org.apache.hadoop.hbase.client.ResultScanner;
+import org.apache.hadoop.hbase.client.RowMutations;
+import org.apache.hadoop.hbase.client.Scan;
+import org.apache.hadoop.hbase.client.Table;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.junit.Assert;
+
+/** Utility class containing various helpers for tests. */
+public class TestUtils {
+
+  public static final String ROW_KEY_PREFIX = "test-row-";
+  public static final byte[] ROW_KEY = "test-row".getBytes();
+  public static final byte[] CF1 = "cf1".getBytes();
+  public static final byte[] CF2 = "cf2".getBytes();
+  public static final TableName TABLE_NAME = TableName.valueOf("replication-test");
+  public static final TableName TABLE_NAME_2 = TableName.valueOf("replication-test-2");
+  public static final byte[] COL_QUALIFIER = "col1".getBytes();
+  public static final byte[] COL_QUALIFIER_2 = "col2".getBytes();
+  public static final String VALUE_PREFIX = "Value-";
+  public static final byte[] VALUE = "Value".getBytes();
+  public static final long TIMESTAMP = 1000l;
+
+  // ONLY STATIC METHODS
```

Comment:
do we need this comment anymore?

Reply:
anymore? Nothing changed, this is a utility class with static method, we explicitly prohibit object creation...